modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -75,19 +75,23 @@ package object dao extends MoreTupleOps with ToUserProgramRoleOps {
       EitherConnectionIO.right[A, T](c)
   }
 
-  // Angle mapping to signed arcseconds via NUMERIC. NOT implicit.
+  // Angle mapping to signed arcseconds via NUMERIC. NOT implicit. We're mapping a type that
+  // is six orders of magnitude more precise than the database column, so we will shift
+  // the decimal point back and forth.
```

Comment:
Maybe the database columns should have more precision?  For offsets that would be overkill, but does this mean we are limited in how precisely an `Angle` can be stored in the database? If there is just one precision for all angles it should probably be the most precise value supported by `Angle`?